### ISSUE TITLE
Tip For Mac Users (Non Issue)

### ISSUE THREAD

**Issue by ALPHAMOTIF** [inkstitch/inkstitch#537]
I recently discovered something handy while using this software on a Mac Computer.  I have found that grouping objects within the SVG file is a crucial step when working with inkstitch.  There are various small quirks that are solved by properly grouping colored objects such as simulate errors etc.  To any mac users struggling with small bugs, try making sure you have grouped like colored elements!

**Comment by patpeazy** [comment]
I have tried this and I still get an error that the path border crosses itself. How do I fix that?

**Comment by ALPHAMOTIF** [comment]
It has to do with your order of operations make sure you ungroup everything then “command option K” or whatever command you have set to “undo” the embroidery grouping for lines, delete all rungs,Go over each shape and make sure that the amount of points is as close to equal as possible for both sides. Re-create the rungs carefully then go piece by piece and group them back to embroidery using “command k” or whatever is set for your computer, then re-group them all and set the color.  Almost every time I run into an error code it’s because I did one little thing somewhere along the line that wasn’t correct whether it’s adding the wrongs and then trying to regroup everything can I just give it to you thank you yeah I’ almost every time I run into an error code it’s because I did one little thing somewhere along the line that wasn’t correct

TAYLOR B.
ALPHAMOTIF
info@alphamotif.net
